Cargo.toml

```diff
@@ -21,5 +21,5 @@ members = [
 	"precompiles/xcm",
 	"precompiles/xvm",
 
-	"contracts/*",
+	# "contracts/*",
```

Comment:
Unintentionally left edit?

Reply:
Added a comment.

It seems that after latest updates, compilation with `cargo check` fails.

`cargo contract check` should work but it also fails for me with another error so I'll have to check that in more detail.

Perhaps this also indicates that we shouldn't keep contracts under root `.toml` file :thinking: